lib/authorized_persona/view_helpers.rb

```diff
@@ -2,9 +2,9 @@ module AuthorizedPersona
   module ViewHelpers
     def authorized_to?(action, resource)
       route = Rails.application.routes.named_routes[resource]
-      raise AuthorizedPersona::Error, "Unable to determine route for #{resource}" if route.nil?
+      raise Error, "Unable to determine route for #{resource}" if route.nil?
```

Comment:
__suggestion: (nitpick)__ This could also be written without the explicit `#nil?` check, eg:

```ruby
raise Error, "Unable to determine route for #{resource}" unless route
```